.github/workflows/ci.yaml

```diff
@@ -114,6 +114,10 @@ jobs:
       fail-fast: false
       matrix:
         strategy: ["pause-and-drain", "progressive", "no-strategy"]
+        case: ["functional"]
+        include:
+        - strategy: "pause-and-drain"
+          case: "ppnd"
```

Comment:
oh, that's cool that you were able to consolidate all of that together.

Reply:
Thank you! After taking a look at Numaflow's CI, I had forgot that you could control the matrix with the `include` field. The reason I had split them into 2 previously is the matrix would have ran our `ppnd` test with each strategy which wouldn't work anyway (and we don't need to do that). This way, we can control specific scenarios much easier.